src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -59,6 +61,7 @@ async function boot(options?: Partial<WebAssemblyStartOptions>): Promise<void> {
   // Configure environment for execution under Mono WebAssembly with shared-memory rendering
   const platform = Environment.setPlatform(monoPlatform);
   Blazor.platform = platform;
+  Blazor.webAssemblyProgressService = new WebAssemblyProgressService;
```

Comment:
Hmm, not sure if TS is just assuming this is a call to the constructor automatically, but should this perhaps be updated to:

```suggestion
  Blazor.webAssemblyProgressService = new WebAssemblyProgressService();
```